src/app-layer-protos.c

```diff
@@ -120,6 +120,9 @@ const char *AppProtoToString(AppProto alproto)
         case ALPROTO_HTTP2:
             proto_name = "http2";
             break;
+        case ALPROTO_HTTP_ANY:
+            proto_name = "http_any";
+            break;
```

Comment:
As Victor mentioned elsewhere on naming..  Should this just output "http", then then HTTP1 and HTTP2 output "http1" and "http2"?

Reply:
Doing this simple change makes S-V test http-connect-fail `for filter {'count': 1, 'match': {'app_proto_orig': 'http', 'event_type': 'flow', 'app_proto': 'dcerpc'}}` because `app_proto_orig` would now be http1
I assumed it is better not to break the API like I did with `AppLayerEventGetProtoByName` in detect-app-layer-event.c